pkg/runtime/runtime.go

```diff
@@ -682,7 +682,12 @@ func (a *DaprRuntime) sendBindingEventToApp(bindingName string, data []byte, met
 		}
 
 		if err != nil {
-			return nil, errors.Wrap(err, "error invoking app")
+			body := resp.Data
+			const charLimit = 75
```

Comment:
Where is this limit coming from? Please, move constant to the top of the file.

Reply:
The reason the limit was 75 is because "error invoking app, body:" is 25 characters. So to keep it at 100 characters, we limit the body to 75 characters. I'm going to change the numbers though, so we can stay consistent with linter's 120 max.